ical.NET/Serialization/iCalendar/Serializers/DataTypes/GeographicLocationSerializer.cs

```diff
@@ -45,9 +46,9 @@ public GeographicLocation Deserialize(string value)
 
             double lat;
             double lon;
-            double.TryParse(values[0], out lat);
-            double.TryParse(values[1], out lon);
-            g.Latitude = lat;
+			double.TryParse(values[0], NumberStyles.Any, CultureInfo.InvariantCulture, out lat);
```

Comment:
Looks weird. It's correct i Visual Studio and Git don't track changes if I redo the indentation on the rows or the hole method. 
Don't know how to fix that if there is a real problem.